### PR TITLE
Cleanup for new website

### PR DESCRIPTION
**Pull request by mathiascode**
I want to create a better page for developers with up-to-date instructions and info about the Soulseek protocol (http://web.archive.org/web/20180216175104/http://www.nicotine-plus.org/wiki/SoulseekProtocol) at some point, but this should be good enough for an initial website.

### CODE REVIEW

**Comment by kiplingw** [comment]
Hey Mat. I don't have any issue with breaking up the markdown into separate files in principal. But a lot of users only look at the README.md when they find the GitHub repository which would, in the case of your proposed PR, remove the information most users are looking for (PPA to just install it).

File: debian/control [hunk 1]
```diff
@@ -15,7 +15,7 @@ Build-Depends:
 Vcs-Git: https://github.com/Nicotine-Plus/nicotine-plus.git
 Vcs-browser: https://github.com/Nicotine-Plus/nicotine-plus
 X-Python-Version: >= 3.5
-Homepage: https://github.com/Nicotine-Plus/nicotine-plus
+Homepage: https://nicotine-plus.org
```
Comment (kiplingw) [inline debian/control:18]: Maybe I missed it, but is old domain configured to load our markdown now?

Comment (mathiascode) [reply debian/control:18]: It will be soon: https://github.com/Nicotine-Plus/nicotine-plus/issues/109#issuecomment-633291564

**Comment by mathiascode** [comment]
I moved the download instructions back into the README.

File: README.md [hunk 2]
```diff
@@ -1,85 +1,52 @@
...
-Please come and join us in the `#nicotine+` channel on Freenode!
-
-If you'd like to contribute, you have a couple of options to get started. You can open an issue ticket on GitHub, discuss in `#nicotine+`, or post to the project [mailing list](mailto:nicotine-team@lists.launchpad.net). Developers are also encouraged to join the [Launchpad Team](https://launchpad.net/~nicotine-team) or subscribe to the mailing list so that they are automatically notified of failed commits.
-
-There is a current list of things [TODO](doc/TODO.md). If you'd like to translate Nicotine+ into another language it has not been already, see [TRANSLATIONS](doc/TRANSLATIONS.md).
-
-You want to contact someone? See: [MAINTAINERS](AUTHORS.md)
-
-# Precompiled Packages
+## Linux
```
Comment (kiplingw) [inline README.md:8]: Please change this to GNU/Linux. Linux is [just a kernel](https://www.gnu.org/gnu/linux-and-gnu.html), after all.

File: doc/TESTING.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Continuous Integration Testing
```
Comment (kiplingw) [inline doc/TESTING.md:1]: We have a lot of developers who fly by and dump PRs sometimes that are not thoroughly tested. I don't think it's a good idea to move this into a separate file, nor most of the other content. The README.md really isn't that large to begin with.

File: setup.py [hunk 4]
```diff
@@ -142,7 +142,7 @@
         license="GPLv3",
         description="Nicotine+, a client for the SoulSeek filesharing network.",
         author="Nicotine+ Contributors",
-        url="https://www.nicotine-plus.org/",
+        url="https://nicotine-plus.org/",
```
Comment (kiplingw) [inline setup.py:145]: Good eye.

**Pull request merged**